sklearn/cluster/_hierarchical.pyx

```diff
@@ -290,13 +290,13 @@ def average_merge(IntFloatDict a, IntFloatDict b,
 
 
 ###############################################################################
-# An edge object for fast comparisons 
+# An edge object for fast comparisons
 
 cdef class WeightedEdge:
     cdef public ITYPE_t a
     cdef public ITYPE_t b
     cdef public DTYPE_t weight
-    
+
```

Comment:
Please check the diff on Github, and remove the changes that are not related to the `np.full` replacement (for instance, additional spaces such as at the line above).

Reply:
I mean reverting changes that were not intentional..